src/main/kotlin/mate/academy/model/FigureList.kt

```diff
@@ -0,0 +1,9 @@
+package mate.academy.model
+
+enum class FigureList {
+        Circle,
+        IsoscelesTrapezoid,
+        Rectangle,
+        RightTriangle,
+        Square
```

Comment:
https://kotlinlang.org/docs/enum-classes.html

reference documentation provides UPPERCASE values for enums